src/GraphQL/Introspection/__Field.cs

```diff
@@ -79,6 +80,32 @@ public __Field()
             Field<NonNullGraphType<BooleanGraphType>>("isDeprecated", resolve: context => (!string.IsNullOrWhiteSpace(context.Source.DeprecationReason)).Boxed());
 
             Field(f => f.DeprecationReason, nullable: true).Description(null);
+
+            FieldAsync<NonNullGraphType<ListGraphType<NonNullGraphType<__AppliedDirective>>>>(
```

Comment:
Is it already approved by graphql specification? https://github.com/graphql/graphql-spec/issues/300 is still open.

As long as I remember additional fields/types in introspection query ruins some playgrounds such as altair.

Reply:
@mikelneo  See `ISchemaFilter.cs` file above and `DefaultSchemaFilter` inside it:
```c#
public virtual Task<bool> AllowType(IGraphType type) => type is __AppliedDirective || type is __DirectiveArgument ? _forbidden : _allowed;
public virtual Task<bool> AllowField(IGraphType parent, IFieldType field) => parent is __Field && field.Name == "directives" ? _forbidden : _allowed;
```